tools/integration_tests/test_config.yaml

```
@@ -72,6 +72,9 @@ write_large_files:
     configs:
       - flags:
           - "--enable-streaming-writes=false"
+          # write-global-max-blocks=2 is for checking multiple file writes in parallel.
```

sorry for the back & forth here but please change it to 5 both in comment and config.
```suggestion
          # write-global-max-blocks=5 is for checking multiple file writes in parallel.
```